src/main/kotlin/no/nav/familie/ef/sak/behandling/revurdering/BehandleAutomatiskInntektsendringTask.kt

```diff
@@ -38,22 +52,42 @@ class BehandleAutomatiskInntektsendringTask(
                 personIdenter = setOf(personIdent),
                 stønadstype = StønadType.OVERGANGSSTØNAD,
             )
+        loggInfoOpprett(personIdent, fagsak)
 
         if (toggle) {
             if (fagsak != null) {
-                secureLogger.info("Kan opprette behandling med $personIdent stønadstype=${StønadType.OVERGANGSSTØNAD} faksakId ${fagsak.id}")
+                loggInfoOpprett(personIdent, fagsak)
```

Comment:
Ønsker man å logge det samme to ganger? 🤔

Reply:
Det er unødvendig, fjernet ene loggingen :)